legacyman_parser/utils/class_html_template.py

```diff
@@ -12,7 +11,8 @@ def __init__(self,
                  av_temp,
                  reduction_ratio,
                  has_tonal,
-                 tonal_href):
+                 tonal_href,
+                 propulsion_href):
```

Comment:
Hello @rnllv - we only call `ClassU` from one place, and in that place we pass a value of `None` for `propulsion_href`.  So, howabout we remove it from the constructor, and set the initial value of `self.propulsion_href` to `None`.

Reply:
Okay